### ISSUE TITLE
[Feature] Add a version upgrade instruction to the go executable file

### ISSUE THREAD

**Issue by ggdream** [golang/go#49088]
Hello, the official team. I have been learning golang for two years now, and I am willing to upgrade to the latest stable version of gopher as soon as possible. So far, the upgrade instructions on the map for each new version released on official Twitter are similar to:

```bash
go install golang.org/dl/go1.17.2@latest
go1.17.2 download
```

If you want to change the installation location, you must modify the source code by the developer. This operation is very troublesome, so every time I delete the original GOROOT, and then unzip the new version to the original GOROOT directory.

In summary, I think if we can add a command such as `go upgrade`?  We all have default or customized GOROOT,  so we can directly overwrite the new file to the GOROOT directory when upgrading.

e.g.
```bash
go upgrade

# output:
# Your GOROOT is : ....
# Process: ....
# Finished it !!
# ${version} is installed in ${GOROOT}
```
